### ISSUE TITLE
Exception running roco outdated

### ISSUE THREAD

**Issue by pauby** [mwallner/rocolatey#122]
When running `roco outdated` I received the following error:

```
[00:00:08] =================================================================================================================     206/206     receive packages from 'chocolatey'
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseError("expected more input")', rocolatey-lib\src\roco\remote.rs:187:58
```

Happy to provide debug info if you let me know what you need.

UPDATE: [Debug info is in this gist](https://gist.github.com/pauby/65d54632511b8cf64b6329e3b9c1b6d6)

**Comment by mwallner** [comment]
heya,
can you reproduce the issue using the verbose (`-v`) parameter? / if possible, please create a gist with the output.

**Comment by pauby** [comment]
Done. See the original comment.

**Comment by mwallner** [comment]
this _should_ be fixed by c6d94b8a73eb7d28a2c7222ce03c1214c006f778
